docs/hub/models-libraries.md

```diff
@@ -39,6 +39,7 @@ The table below summarizes the supported libraries and their level of integratio
 | [Timm](https://github.com/rwightman/pytorch-image-models)                   | Collection of image models, scripts, pretrained weights, etc.                        | ✅ | ✅ | ✅ | ✅ |
 | [Transformers](https://github.com/huggingface/transformers)                 | State-of-the-art Natural Language Processing for Pytorch, TensorFlow, and JAX        | ✅ | ✅ | ✅ | ✅ |
 | [Transformers.js](https://github.com/xenova/transformers.js)                | State-of-the-art Machine Learning for the web. Run 🤗 Transformers directly in your browser, with no need for a server! | ❌ | ❌ | ✅ | ❌ |
+| [Unity Sentis](https://unity.com/products/sentis)                | Inference engine for the Unity 3D game engine | ❌ | ❌ | ✅ | ✅ |
```

Comment:
Actually, there's no library specific way to download or push sentis models to the Hub, is there?

Reply:
Library specific? Not yet I don't think. So is it four X's?

Reply:
I changed it to XXXX